### ISSUE TITLE
Haze of Evil (tracker category)

### ISSUE THREAD

**Issue by Painstormx** [Questie/Questie#4952]
In Classic Era:.
This quest: https://www.wowhead.com/classic/quest=4143/haze-of-evil
In the quest tracker it is put in a "The Temple of Atal'Hakkar" category.
But all the other Sunken Temple quests are in a "Sunken Temple" category.
Furthermore, in the Blizzard quest log, this quest is actually in the "Un'goro Crater" category.
Would make most sense if the quest was put in "Sunken Temple" in the tracker along with all the quests that take place there.

**Comment by Dyaxler** [comment]
The Quest is placed into a "Zone" aka Category based on the Zone ID or Name embedded in the QuestDB entry... this should probably be better handled via a Quest Correction.

**Comment by Painstormx** [comment]
Yeah, it is kinda strange if the Zone ID comes back as "The Temple of Atal'Hakkar" since Blizzard themselves don't use that in the quest log either. I guess the other quests in that dungeon, like "Secret of the Circle" have different Zone IDs for some reason, since they are in the "Sunken Temple" zone in the quest log.

**Comment by Dyaxler** [comment]
Some quests are in the temple but "outside" the instance.

**Comment by Cabro** [comment]
We will just set the category as it shows in the quest log. Most likely this is the problem OP is reporting

**Comment by Dyaxler** [comment]
Meh... we can just find all the quests that are inside and around Sunken Temple and change the zoneID's to "Sunken Temple". That would put them all in the same "Zone" name.

I.E. change all the zoneOrSort keys to 1417 ( `QuestieDB:GetQuest(questID).zoneOrSort` )

![image](https://github.com/Questie/Questie/assets/17400948/549f622d-a680-4900-83ee-52413e4d7a53)

**Comment by Painstormx** [comment]
I know I have absolutely no saying in this, but I agree it would make it much easier for players if the quests were grouped by where they actually take place, instead of where they start as is how Blizzard often handle them.
Sunken Temple is especially bad as the quests in the dungeon seems to be labelled as either "Sunken Temple", "The Temple of Atal'Hakkar", "Un'goro Crater" and "Stormwind City" (Blizzards choice, I know).

**Comment by MyTechnoHunter** [comment]
personally, i cheat.. 
i use Atlas/Atlasloot/AtlasQuest addons to find out where and what quests are for doing in the dungeon.. would be nice if Questie could group em.. i agree

**Comment by Cabro** [comment]
There are maaaaaaaaany dungeons that have quests in all sort of categories. Just stick to what the quest log shows you and you're golden?

**Comment by MyTechnoHunter** [comment]
could add a new sort option, dungeon? make it a feature request?

**Comment by MyTechnoHunter** [comment]
on the other hand.. dont we sort of assume people are gonna hide the tracker in dungeons? -ponder-

**Comment by Dyaxler** [comment]
Dungeon sort? NOOOOOO.... then we'd have to custom make a Zone Sort title for EACH dungeon in the Tracker Sort Function. HELL NO!

Better to just update the zoneOrSort tags as we go. Little by little. Besides... you can untrack all your quests and just track the ones for that dungeon. Or minimize all the other Zones.

A player can still opt to hide the Tracker in a dungeon. If the quests they need to do is "outside" the dungeon at least they will all be grouped together in one Zone like @Painstormx suggests. And I agree with them.
